Packs/CommonTypes/IndicatorTypes/reputation-Onion_Address.json

```diff
@@ -12,5 +12,6 @@
     "system": false,
     "updateAfter": 0,
     "version": -1,
-    "fromVersion": "5.0.0"
+    "fromVersion": "5.0.0",
+    "layout": "domainRepUnified"
```

Comment:
Onion address - should have the domain layout?

Reply:
it is a domain, we don't have a layout for it. It's either that or no layout which is exactly the same (only seems like bad practice to me).